posts/2021-01-22-ee9-messaging-security-21002-beta.adoc

```diff
@@ -0,0 +1,205 @@
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.2-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.2-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have a number of new API Release Candidates to join the expanding library of supported Jakarta packages.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* <<messaiging, Messaging 3.0 (`messaging-3.0, messagingClient-3.0, messagingServer-3.0, messagingSecurity-3.0`)>>
+* <<security, Security 2.0 (`appSecurity-4.0, appSecurityClient-1.0`)>>
+* Batch 2.0 (`batch-2.0`)
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta WebSocket 2.0 (`websocket-2.0`; now with full CDI integration)
+* RESTful Web Services 3.0 (`restfulWS-3.0` and `restfulWSClient-3.0`)
+* Jakarta Server Faces 3.0 (`faces-3.0`)
+* Jakarta Connectors 2.0 (`connectors-2.0`)
+* Jakarta Enterprise Beans 4.0 (`enterpriseBeans-4.0`)
+* Jakarta Enterprise Beans Remote 4.0 (`enterpriseBeansRemote-4.0`)
+* Jakarta Enterprise Beans Home 4.0 (`enterpriseBeansHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`enterpriseBeansLite-4.0`)
+* Jakarta EE Application Client 9.0 (`javaeeClient-9.0`)
+* Jakarta Authentication 2.0 (`jaspic-2.0`)
+* Jakarta Authorization 2.0 (`jacc-2.0`)
+* Jakarta Persistence 3.0 (includes Eclipselink 3.0-RC1.) (`jpa-3.0`)
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Beans 2.0 (`managedBeans-2.0`)
+* Jakarta Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Bean Validation 3.0 (`beanValidation-3.0`)
+* Jakarta Contexts and Dependency Injection 3.0 (`cdi-3.0`)
+* Message-Driven Beans 4.0 (`mdb-4.0`)
+* JDBC 4.2 & 4.3 (`jdbc-4.2` & `jdbc-4.3`)
+* Jakarta Transactions 2.0 (`transaction-2.0`)
+* Jakarta JSON Binding 2.0 (`jsonb-2.0`)
+* Jakarta JSON Processing 2.0 (`jsonp-2.0`)
+* Jakarta Servlet 5.0 (`servlet-5.0`)
+* Jakarta Server Pages 3.0 (`jsp-3.0` now `pages-3.0`)
+* Jakarta Expression Language 4.0 (`el-4.0` now `expressionLanguage-4.0`)
```

Comment:
Jakarta Mail ('mail-2.0") is missing from the list

Reply:
The ones that say xxx-y.z now xxxx-y.z have been the new name for a while.  Should we just have the new name now?

Reply:
Mail added and removed old names.

Reply:
You have "Jakarta Jakarta Mail".  It should just be Jakarta Mail.

Reply:
Fixed.